grid/example/src/App.js

```diff
@@ -361,7 +361,7 @@ const App = memo(() => {
                 Cell: ({ row }) => {
                     return (
                         <div className="action">
-                            <RowOptions row={row} selectRowOptions={selectRowOptions} />
+                            <RowOptions updateCellData={updateCellData} row={row} airportCodeList={airportCodeList} selectRowOptions={selectRowOptions} />
```

Comment:
I think we can remove selectRowOptions (this just console logs the data) method from this file and the rowoptions.js file, as we are going to implement edit/delete/pin rows functionalities.

Reply:
removed selectRowOptions from all occurrences and RowOptions.js